### PR TITLE
Doxygen Comments

### PR DESCRIPTION
**Pull request by lesliechase**

### CODE REVIEW

**Comment by yiskylee** [comment]
@lesliechase , did you pull the develop first?

**Comment by lesliechase** [comment]
I think this should be all set now.

File: cpp/include/cpu_operations.h [hunk 1]
```diff
@@ -51,6 +51,20 @@ class CpuOperations {
...
+  /// Input Matrix
+  /// \param b
+  /// Input Scalar
+  ///
+  /// \return
+  /// This function returns a matrix that is the resulatant of adding the
+  /// input matrix and scalar.
+  ///
+  /// \sa
+  /// \ref Add
```
Comment (yiskylee) [inline cpp/include/cpu_operations.h]: ref should be used for other functions that are related to this one. If I click Add in the webpage, it actually just goes to the same function Add. Is there some way to actually ref the Add function that takes two Matrices here?

File: cpp/include/cpu_operations.h [hunk 2]
```diff
@@ -62,6 +76,20 @@ class CpuOperations {
...
+  /// Input Matrix 1
+  /// \param b
+  /// Input Matrix 2
+  ///
+  /// \return
+  /// This function returns a matrix that is the resulatant of adding two
+  /// input matricies.
+  ///
+  /// \sa
+  /// \ref Add
```
Comment (yiskylee) [inline cpp/include/cpu_operations.h]: Same thing here, it would be great if we can have two Add functions refer to each other.

**Pull request merged**